Formula/l/llvm.rb

```diff
@@ -445,10 +463,28 @@ def install
     end
   end
 
+  def post_install
+    return unless OS.mac?
+    return unless (macos_sdk = MacOS.sdk_path_if_needed)
```

Comment:
Note that this is best effort and might point to a newer SDK if CLT is not installed which wouldn't be great.

Reply:
Fair enough, but note this is the path we set `DEFAULT_SYSROOT` to so this is strictly an improvement over existing builds (because users can now fix a broken `--sysroot` flag when previously they couldn't).

Reply:
It's a relocatable bottle. Anybody building from source are long past the days of updating Xcode and are required by brew to have the CLT installed.

Reply:
So should we just require the CLT regardless?

Reply:
Probably